### ISSUE TITLE
Fix the maximise window resizing.

### ISSUE THREAD

**Issue by lee-halford** [MicrosoftDocs/WDAC-Toolkit#353]
Feature enhancement suggestion.

When maximizing the window, the contents of the window do not resize with it. 

How easy would it be to have the contents also resize, like the adding policy rules scrolling window resize so you can see more on the screen at once?

**Comment by a-shina** [comment]
@jgeurten, I think it would be great to have the contents resize when the window is maximized.

![image](https://github.com/user-attachments/assets/5e7c9951-ef89-4a65-97f5-24a39bbab71d)

**Comment by JackStuart** [comment]
Yeah this would be awesome, I just spent most of the day trying to figure it out but couldn't make it work. I did manage to reorder the columns so that the policyname is last and make the column size user-adjustable though so will probably just run that. There's no bar down the bottom so you can actually see everything at the same time 

<img width="556" alt="image" src="https://github.com/user-attachments/assets/327ca871-5b2a-419c-a351-a9f84e9bc780">
